data/lark

```diff
@@ -0,0 +1,7 @@
+# Lark
+larksuite.com
+larksuitecdn.com
+
+# Feishu
+feishu.cn
+feishucdn.com
```

Comment:
"feishu" belong "lark" ?

Reply:
https://en.wikipedia.org/wiki/Lark_(software) different name in different region